release/ray_release/cluster_manager/cluster_manager.py

```diff
@@ -30,7 +30,7 @@ def __init__(
         self.cluster_compute_name = None
         self.cluster_compute_id = None
 
-        self.autosuspend_minutes = 10
+        self.autosuspend_minutes = 120
```

Comment:
I think this is less prone to error setting this higher since this feature doesn't seem to be that stable

Reply:
Did we observe any errors here previously? I'd like to keep it as low as possible as cluster shutdown is not very stable for failing tasks and can cost us a lot of money

Reply:
Yes. It seems like there's the prod side bug that will not detect the job-based workloads properly.

Reply:
> cluster shutdown is not very stable for failing tasks

Can you tell me a bit more about it?

Reply:
To tackle the job thing, let's set the autosuspend minutes for the respective tests then individually until the bug is fixed on the product side. Also it seems like this should be prioritized on their end

Reply:
Sorry I expressed myself incorrectly. Cluster shutdown for failing tasks is actually quite stable, but it sometimes fails when Buildkite jobs are aborted (e.g. using "Cancel"). In those cases, the cluster would still come up and stay up for 2 hours.
We've also seen that cluster startup timed out in the past and the cluster came up later and ran until autosuspend triggered. This should have been fixed, but there are no guarantees or integration tests for that.
Generally, the autosuspend is a central tool for us to ensure that we don't waste a lot of money on release tests, so I'd rather urge the procuct team to fix these detection bugs than disable this fallback.

Reply:
Hmm isn’t this premature cost optimization? I think it’s more stable migration process to make it same as before (120m was the default for old infra) and decrease it after it is done. 

 What’s the exact scenario that autosuspend will help? Are we worried that the new code base doesn’t shutdown the cluster properly because of bugs in our end?

Reply:
Basically I’d like to suggest we should focus stable migration first and apply cost optimization later. Also based on what you said the edge cases don’t seem to happen commonly and the benefit of cost saving seems marginal. Autosuspend feature also doesn’t seem to be very stable feature

Reply:
As discussed offline, fine with that. Let's converge to min(120, command_timeout + 10) though